setup.py

```diff
@@ -29,24 +28,27 @@
     ".bmp",
 ]
 
+ROOT = Path(os.getcwd()).parent
+
 def main():
-    root = Path(os.getcwd()).parent
-    os.chdir(root)
-    for dirpath, _, files in os.walk(root):
+    os.chdir(ROOT)
+    for dirpath, _, files in os.walk(ROOT):
         if is_ignored_folder(dirpath):
             continue
         for file in files:
             if is_ignored_extension(file):
                 continue
             replace_file_content(os.path.join(dirpath, file))
             rename_file(file, dirpath)
-    rename_folders(root)
+    rename_folders(ROOT)
```

Comment:
Como você tirou o argumento da função, precisa tirar o argumento na hora de invocar

```suggestion
    rename_folders()
```